benchmarks/src/main/java/com/slack/kaldb/QueryBenchmark.java

```diff
@@ -1,169 +0,0 @@
-package com.slack.kaldb;
-
-import brave.Tracer;
-import brave.Tracing;
-import com.google.protobuf.ByteString;
-import com.slack.kaldb.logstore.LogMessage;
-import com.slack.kaldb.logstore.LuceneIndexStoreImpl;
-import com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl;
-import com.slack.kaldb.logstore.search.LogIndexSearcher;
-import com.slack.kaldb.logstore.search.LogIndexSearcherImpl;
-import com.slack.kaldb.logstore.search.aggregations.DateHistogramAggBuilder;
-import com.slack.kaldb.writer.LogMessageWriterImpl;
-import com.slack.service.murron.Murron;
-import io.micrometer.core.instrument.MeterRegistry;
-import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.text.SimpleDateFormat;
-import java.time.Duration;
-import java.util.Comparator;
-import java.util.Random;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.logging.Logger;
-import java.util.stream.Stream;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.common.record.TimestampType;
-import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.annotations.Level;
-import org.openjdk.jmh.annotations.Scope;
-import org.openjdk.jmh.annotations.Setup;
-import org.openjdk.jmh.annotations.State;
-import org.openjdk.jmh.annotations.TearDown;
-
-@State(Scope.Thread)
-public class QueryBenchmark {
```

Comment:
Why is benchmark code deleted as part of this change?

Reply:
good question! We'll be brining them back once some of our current schema work is done. We haven't used these in years and some of it needs to be redone anyways

We also want to remove stuff like api log / murron and other slack specific constructs before our GA